### PR TITLE
Sequential message sending

### PR DESCRIPTION
**Pull request by RockChuLee**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

Add the English-version Delay Message Sending of Producer.

## Brief changelog

Add 02-Producer/06message2.md

## Verifying this change

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by caigy — CHANGES_REQUESTED** [review]
In current official site, 'ordered message' is used. It's difficult to tell 'ordered message' or 'sequential message' is more accurate, but I think we should keep the current term for consistency.

**Comment by RockChuLee** [comment]
> In current official site, 'ordered message' is used. It's difficult to tell 'ordered message' or 'sequential message' is more accurate, but I think we should keep the current term for consistency.

Has been modified.

File: docs/09-英文/02-Producer/06message2.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Ordered messages have strict requirements on the order in which they are sent and consumed. 
+
+For a given topic, messages are published and consumed strictly on a first-in-first-out (FIFO) basis, messages published first are consumed first and messages published later are consumed later. Partitioned ordered messages are supported in Apache RocketMQ, as shown in the following figure. We can partition messages according to a certain criterion (e.g., the ShardingKey in the figure). Messages with the same ShardingKey are assigned to the same queue and consumed in order.
+
+![顺序消息发送](../../picture/顺序消息发送.png)
+
+Ordered messages are also used in a wide range of application scenarios, such as in the example of creating an order, where you need to ensure that the same order is generated, paid and shipped, and these three operations are executed sequentially. In the case of normal messages, the messages of order A may be polled and sent to different queues, and the messages of different queues will not be able to maintain the order while ordered messages are sent by routing the sequence of messages with the same ShardingKey (same order number) to a logical queue.
```
Comment (tsunghanjacktsai) [inline docs/09-英文/02-Producer/06message2.md:10]: > Ordered messages are also used in a wide range of application scenarios, such as in the example of creating an order, where you need to ensure that the same order is generated, paid and shipped, and these three operations are executed sequentially. In the case of normal messages, the messages of order A may be polled and sent to different queues, and the messages of different queues will not be able to maintain the order while ordered messages are sent by routing the sequence of messages with the same ShardingKey (same order number) to a logical queue.

to: 

> Sequential messages are also used in a wide range of application scenarios, such as the example of creating orders, the same order generation, payment, and shipment should be executed sequentially. In the case of simple messages, the messages of Order A may be polled and sent to different queues. The messages of different queues will not be able to maintain order. In contrast, ordered messages are sent by routing the sequence of messages with the same ShardingKey (same order number) to a logical queue.

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
@RockChuLee

File: docs/09-英文/02-Producer/06message2.md (hunk 1 above)
Comment (RockChuLee) [reply docs/09-英文/02-Producer/06message2.md:10]: > In current official site, 'ordered message' is used. It's difficult to tell 'ordered message' or 'sequential message' is more accurate, but I think we should keep the current term for consistency.

According to this rule, should we use "Ordered messages" to replace "Sequential messages" at begin of the paragraph?

Comment (tsunghanjacktsai) [reply docs/09-英文/02-Producer/06message2.md]: Ahh yes, my mistake. "Ordered messages" is correct.

**Pull request merged**